docs_source_files/content/webdriver/driver_requirements.en.md

```diff
@@ -632,4 +632,25 @@ so whilst the web will continue to change, PhantomJS will not be updated.
 This was after Google announced the ability to run Chrome headlessly, 
 something also now offered by Mozilla's Firefox.
 
+### Chocolatey
 
+On Windows systems the driver binaries can be installed individually, or as a collection using [chocolatey](https://chocolatey.org/).
+
+* To find the available drivers, use choco list:
```

Comment:
Let's avoid adding the commands here, we should link to their official page that shows how to use the packages, if we duplicate things here, they can get easily out of date.